### DISCUSSION TITLE
Why take down release twice

### DISCUSSION THREAD

**Discussion by xlionjuan** [rustdesk/rustdesk#6070] [General]
I found out 1.2.2 and 1.2.3 released and take down after 1 day, it is a serious problem with trust and cause issues on community maintained software list(e.g. Homebrew)
I hope you can be more meticulous with releases.

**Comment by deep-soft** [comment]
You can find all releases here:
https://github.com/rustdesk/rustdesk/releases

**Reply by xlionjuan** [reply]
I know, please see the title.
